medcat/utils/ner/deid.py

```diff
@@ -62,18 +62,73 @@ def train(self, json_path: Union[str, list, None],
               *args, **kwargs) -> Tuple[Any, Any, Any]:
         return super().train(json_path, *args, train_nr=0, **kwargs)  # type: ignore
 
-    def deid_text(self, text: str, redact: bool = False) -> str:
+    def get_chunks(self, text:str, de_id_pipe,maximum_tkns) -> List[List[Any]]:
```

Comment:
A few more things.
1. Please add type hints to arguments as well (and the linting issue).
2. The method should also have a docstring.
3. This method only depends on the tokenizer, not the entire pipe
4. I think this method should probably be extracted from the class, probably into its own module (i.e `chunking`) in the same package. After all, the method does not depend on the instance at all.
4. This method should simply return a list (or better yet, a generator) of strings (i.e the text chunks), not some obscure data structure.